protocol-designer/src/pages/Designer/ProtocolSteps/Timeline/StepContainer.tsx

```diff
@@ -200,7 +200,7 @@ export function StepContainer(props: StepContainerProps): JSX.Element {
         gridGap={SPACING.spacing4}
         flexDirection={DIRECTION_COLUMN}
       >
-        <Btn
+        <Box
```

Comment:
because this has `onClick`

```suggestion
        <Box
          role="button"
```